README.md

```diff
@@ -36,6 +36,16 @@ user_validator:
 github:
   timeout: Timeout in seconds for Github request (default: 60s)
 
+gitlab:
+  glBaseURL: GitLab instance base url
+  glUsername: GitLab username associated with token
+  glToken: Token to access GitLab. Must have repository read permission
```

Comment:
These settings are also under `git_partition_sync_producer` as of now.  See: https://github.com/app-sre/go-qontract-reconcile/blob/main/internal/gitpartitionsync/producer/producer.go#L61